.github/workflows/sudo-python-tests.yml

```diff
@@ -88,6 +88,17 @@ jobs:
           check_name: Sudo Python test results
           junit_files: lte/gateway/test-results/**/*.xml
           check_run_annotations: all tests
+      - name: Publish results to Firebase
+        if: always() && github.event.workflow_run.event == 'push'
+        env:
+          FIREBASE_SERVICE_CONFIG: ${{ secrets.FIREBASE_SERVICE_CONFIG }}
+          REPORT_FILENAME: "sudo_python_tests_${{ env.SHA }}.html"
+        run: |
+          npm install -g xunit-viewer
+          [ -d "lte/gateway/test-results/" ] && { xunit-viewer -r lte/gateway/test-results/ -o "$REPORT_FILENAME"; }
+          [ -f "$REPORT_FILENAME" ] && { python ci-scripts/firebase_upload_file.py -f "$REPORT_FILENAME" -o out_url.txt; }
+          [ -f "out_url.txt" ] && { URL=$(cat out_url.txt); }
```

Comment:
Can we not just omit the recurring file exist checks here, if we use the `&&` connection? At least I would believe so.

And if you output not to `out_url.txt` but to standard out (presumably by omitting the `-o` :shrug: ) you can even do this with one line less. Something like `URL=$(python ci-scripts/firebase_upload_file.py -f "$REPORT_FILENAME")`.

Reply:
Second thought:
I cannot believe that we have two self made scripts, which do not work together at all. While this should imho be properly cleaned up inside, I see that this might be out of scope of this ticket. Unless you want to do a bit of Python coding. :wink:

Reply:
I agree, this should be cleaned up in a follow up PR to merge the overall functionality to master earlier.